helm/flink/templates/deployment-jobmanager.yaml

```diff
@@ -10,6 +10,16 @@ spec:
         app: {{ template "fullname" . }}
         component: jobmanager
     spec:
+      initContainers:
+      - name: flink-copy-libs
+        image: {{ .Values.image }}:{{ .Values.imageTag }}
```

Comment:
The init container should probably also respect `imagePullPolicy`. (also in `deployment-taskmanager.yaml`)